arch/risc-v/src/mpfs/Kconfig

```diff
@@ -157,6 +157,13 @@ config MPFS_DDR_TYPE
 	default 3 if MPFS_DDR_TYPE_LPDDR3
 	default 4 if MPFS_DDR_TYPE_LPDDR4
 
+config MPFS_ENABLE_CACHE
+	bool "Enable L2 cache"
+	depends on MPFS_BOOTLOADER
```

Comment:
@jlaitine why does it depend on MPFS_BOOTLOADER ?

Reply:
The MPFS_BOOTLOADER option is used when building a nuttx image which does memory and clock configuration (i.e. boots stand alone from power on reset). If one uses some external bootloader (HSS, another nuttx instance or such), the cache enablement needs to be done in that one. Especially if running multiple OS's in parallel, it only makes sense to configure the cache in the boot. Also, cache enablement can only be done on the E51 hart, and not from anything running on U54's. Original code enabled the L2 cache always with MPFS_BOOTLOADER. This change only makes it possible to also disable the cache.